content/blog/rueckblick-spieleentwickler-treffs/contents.lr

```diff
@@ -6,7 +6,7 @@ Seit Mitte 2018 finden in der Toolbox die Spieleentwickler-Treffen von [Spieleen
 
 Im folgenden ein kurzer Rückblick über die vergangenen Treffen.
 ---
-pub_date: 2019-02-07
+pub_date: 2019-02-06
```

Comment:
Change not relevant for this PR

Reply:
Der validator beschwert sich über das doppelte exakt gleiche datum. also habe ich das geändert.

Reply:
Ist aber nicht "XML Feed syntax"...

Reply:
... ja